tools/Crank/BenchmarkApps/Driver/Controllers/Invoke.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace Driver.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class Invoke : ControllerBase
+    {
+        private IJobHost _jobHost;
+
+        public Invoke(IJobHost jobHost)
+        {
+            _jobHost = jobHost;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Get(string scenario)
```

Comment:
Single invoke entrypoint that direct invokes different job functions based on "scenario" param